pjmedia/src/pjmedia-audiodev/errno.c

```diff
@@ -85,7 +85,8 @@ PJ_DEF(pj_str_t) pjmedia_audiodev_strerror(pj_status_t statcode,
         PJ_UNUSED_ARG(ca_err);
         // TODO: create more helpful error messages
         errstr.ptr = buf;
-        pj_strcpy2(&errstr, "Core audio error");
+        pj_str_t msg = pj_str("Core audio error");
```

Comment:
only minor, but this declaration needs to be moved above, in order not to trigger any warning/failure with the old compiler standard

Reply:
Not sure I understand. Do you mean `pj_str_t msg = pj_str("Core audio error");` should go below `pj_str_t errstr;` in line 74?

Reply:
No, it should go to L84 only